game-core/src/main/java/games/strategy/triplea/ui/mapdata/MapData.java

```diff
@@ -354,6 +356,14 @@ public int getDefaultUnitsStackSize() {
     return Math.max(0, Integer.parseInt(stack));
   }
 
+  public boolean shouldDrawUnit(final String unitName) {
+    if (undrawnUnits == null) {
+      final String property = mapProperties.getProperty(PROPERTY_DONT_DRAW_UNITS, "");
+      undrawnUnits = new HashSet<>(Arrays.asList(property.split(",")));
```

Comment:
Huh, I just checked the build log on this branch, and this line doesn't trigger an Error Prone StringSplitter warning.  I wonder what exactly pushes Error Prone over the edge to raise a warning? :confused: 

For future reference, when you need to split a string into another container that requires a `Collection` (like `HashSet` does), you can avoid the intermediate array with, for example, `new HashSet<>(Splitter.on(',').splitToList(property))`.

Reply:
Yeah, I've seen your couple of PRs on using Splitter but I just mirrored this code to be similar to how a similar property was doing it and didn't see any warnings on them so decided to leave it.